Assignment2-Python/sumLists.py

```diff
@@ -0,0 +1,92 @@
+#2.5 Sum Lists (p. 95)
+
+class Node:
+    def __init__(self, data=0, next=None):
+        self.data = data
+        self.next = next
+
+def print_list(node):
+    while node:
+        print(node.data, end=" -> ")
+        node = node.next
+    print("None")
+
+def sum_lists_reverse(l1, l2):
+    carry = 0
+    dummy_head = Node()
+    current = dummy_head
+
+    while l1 or l2 or carry:
+        total = carry
+        if l1:
+            total += l1.data
+            l1 = l1.next
+        if l2:
+            total += l2.data
+            l2 = l2.next
+        carry = total // 10
+        current.next = Node(total % 10)
```

Comment:
This might not be as useful since you're already passing `total % 10` directly to the `Node` constructor, but Python does have a [divmod](https://www.w3schools.com/python/ref_func_divmod.asp) function that will calculate both the quotient and remainder of two numbers.